index.html

```diff
@@ -4,83 +4,219 @@
-								<div class="form-group">
-									<label class="sr-only" for="exampleInputEmail2">Email address</label>
-									<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
-									<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+							<form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+								target="_blank" novalidate>
+								<div class="row">
+									<div class="form-group">
+										<input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+										<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+									</div>
+									<button type="submit" class="btn btn-first btn-email btn-fill">NOTIFY ME</button>
 								</div>
-								<button type="submit" class="btn btn-email btn-fill">Notify Me</button>
-
 							</form>
-							<div class="media">
-								<a href="#">
+
+							<div class="media mb-3">
+								<a href="https://www.linkedin.com/company/virtision-com">
 									<i class="fa fa-linkedin-square"></i>
 								</a>
 
 								<a href="#">
 									<i class="fa fa-facebook"></i>
 								</a>
 
-								<a href="#">
-									<i class="fa fa-envelope-o"></i>
+								<a href="mailto:team@virtision.com">
+									<i class="fa fa-envelope"></i>
 								</a>
 
 							</div>
 						</div>
 					</div>
 				</div>
+				<div class="col-lg-5 offset-lg-1 float-right mb-3">
+					<video playsinline="" muted="" class="rounded shadow-lg video" src="images/demo.mp4" loop autoplay></video>
+				</div>
+
+			</div>
+		</div>
+	</div>
+	<div class="main">
+		<div class="container value">
+			<div class="row justify-content-center">
+				<div class="col-lg-8 center">
+					<h1 class="pop"> For any creatives at any level to create and share their imagination in XR</h1>
+					<!--Toy Faces created by Amrit Pal Singh (https://amritpaldesign.com/toy-faces)-->
+				</div>
+				<div class="col-lg-12">
+					<img style="width:inherit" class="rounded shadow-lg" src="./images/artist2.png">
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row pt-5">
+				<div class="col-lg-6">
+					<img class="demo" src="./images/creation.png" width="400">
+				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Simply create your XR story with just a few taps</h1>
+					<h3 class="motto">experience a unified and simplified creation process to unblock your creativity</h3>
```

Comment:
This is good. Just another wording tweak. 

"Our seamless creation process will unblock your creativity."
"Use our unified creation process to unblock your creativity."
"Our simplified creation process won't block your creativity."

Reply:
I decide to change to this: "our seamless creation experience will unblock your creativity."